app/helpers/partners_helper.rb

```diff
@@ -14,4 +14,18 @@ def show_header_column_class(partner, additional_classes: "")
       "col-sm-4 col-4 #{additional_classes}"
     end
   end
+
+  def humanize_boolean(boolean)
+    boolean ? 'Yes' : 'No'
+  end
+
+  def partner_status_badge(partner)
+    if partner.partner_status == "verified"
+      tag.span partner.partner_status, class: %w(badge badge-pill badge-primary float-right)
+    elsif partner.partner_status == "recertification_required"
+      tag.span partner.partner_status, class: %w(badge badge-pill badge-danger float-right)
+    else
+      tag.span partner.partner_status, class: %w(badge badge-pill badge-info float-right)
```

Comment:
any thoughts on consolidating the common classes in an array at the beginning and then adding the badge-primary/badge-danger/badge-info per case?